components/forward_post_modal/forward_post_comment_input.tsx

```diff
@@ -151,6 +152,8 @@ const ForwardPostCommentInput = ({channelId, canForwardPost, comment, permaLinkL
             useChannelMentions={false}
             supportsCommands={false}
             suggestionListPosition='bottom'
+            suggestionList={RhsSuggestionList}
```

Comment:
i use it here since this automatically switches position when the list would otherwise be rendered outside of the viewport ... As mentioned by @hmhealey [here](https://github.com/mattermost/mattermost-webapp/pull/11030#issuecomment-1234551322) I will add this feature to the normal SuggestionList instead and remove this wrapper component completely

Reply:
I did remove it for now since it is not really needed here and I will merge the two components (as described in [this ticket](https://mattermost.atlassian.net/browse/MM-46850))